### ISSUE TITLE
🐛 Sentry error: NoMethodError:Hyrax::AttachmentsController#manifest

### ISSUE THREAD

**Issue by jillpe** [notch8/utk-hyku#446]
# Summary
related to: 
- #442 

csv: 

[rfta_test_with_attachments_video.csv](https://github.com/scientist-softserv/utk-hyku/files/11872075/rfta_test_with_attachments_video.csv)

[Import URL](https://dc.utk-hyku-staging.notch8.cloud/importers/134?locale=en)

When trying to import the following csv. All of the jobs ran successfully except the create relationships, and it resulted in this sentry error: https://scientist-inc.sentry.io/issues/4276294784/?project=6745049&query=is%3Aunresolved&referrer=issue-stream&stream_index=0

# Accepted Criteria
- [ ] Accessing the /manifest.json of the above work does not error out

**Comment by ShanaLMoore** [comment]
### Dev Notes 

CreateRelationshipsJob is failing with the following error, in staging and locally, so this error is caused by no fileset being present: 


```
Ldp::BadRequest: Prefix collections has not been registered
```

Maybe helpful? https://github.com/samvera/ldp/blob/303c1bd620372e4438252833f2f5115742e1adea/spec/lib/ldp/resource_spec.rb#L36-L45

it appears to be coming from [here](https://github.com/samvera-labs/bulkrax/blob/main/app/jobs/bulkrax/create_relationships_job.rb#L59)

Stepping into the above, in particular it breaks [here](https://github.com/samvera-labs/bulkrax/blob/main/app/models/concerns/bulkrax/dynamic_record_lookup.rb#L21)

```
record ||= ActiveFedora::Base.find(identifier) => Ldp::BadRequest: Prefix collections has not been registered
from /usr/local/bundle/gems/ldp-1.0.3/lib/ldp/client/methods.rb:118:in `block in check_for_errors'
```

identifier: "collections:rftatest"

TODO: 

add guard to handle such errors, so that it doesn't crash

UPDATE: The above is a a strange one. This happened because the original CSV linked in this ticket has a record that references parent identifiers that don't exist. When I removed the parent reference the job proceeded with success. 

altered file: 


[rfta_test_with_attachments_video(1).csv](https://github.com/scientist-softserv/utk-hyku/files/11884565/rfta_test_with_attachments_video.1.csv)
